bin/run-reference-solution-tests-in-docker.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+
+# Synopsis:
+# Test the test runner Docker image by running it against all reference solutions from the Groovy repository.
+# It is recommended to run this script when upgrading Groovy exercises or Groovy test runner dependencies.
+
+# Output:
+# Fails is
+
+# Example:
+# ./bin/run-reference-solution-tests-in-docker.sh
+
+# Build the Docker image
+docker build --rm -t exercism/groovy-test-runner .
+export BUILD_IMAGE=false
+
+# Assumes exercism/groovy repository is checked out next to the exercism/groovy-test-runner repository.
+# If the repository is checked out elsewhere, provide the path as the first argument.
+groovy_repo_path=$(realpath "${1:-${PWD}/../groovy}")
```

Comment:
Is `realpath` standard? I'm used to `readlink -e`

Reply:
I used the same command as the existing `run-tests.sh`: https://github.com/artamonovkirill/groovy-test-runner/blob/86676ee7822a6bb22b9a366887a8383bf4d10137/bin/run-tests.sh#L19

Reading [this thread](https://unix.stackexchange.com/questions/136494/whats-the-difference-between-realpath-and-readlink-f), `readlink` might have been more widely available in 2014, but I am not sure about now.

I'd opt for `realpath` for consistency.

Reply:
Also, seems like `readpath` is used more often in the Exercism repos:
- `realpath`: https://github.com/search?q=owner%3Aexercism+realpath+language%3AShell&type=code
- `readlink`: https://github.com/search?q=owner%3Aexercism+readlink+language%3AShell&type=code